docs/get-started/control-motor.md

```diff
@@ -244,6 +244,197 @@ func main() {
 }
 ```
 
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+Flutter code must be launched from inside a running Flutter application.
+To get started programming your rover with Flutter, follow the instructions to [Build a Flutter App that Integrates with Viam](/tutorials/control/flutter-app/).
```

Comment:
I would not make them go through this entire tutorial. They don't need a locations screen etc. for a quickstart. https://docs.viam.com/get-started/drive-rover/ can also be simplified. When you tested this code, did you go through the whole flutter app tutorial or were you able to do a simpler subset of those instructions? I believe I've made test apps where I made the "home" screen after login just be the machine screen with its resources, and then make one or more of them navigable, so they auth to one machine but don't need to build the org and location screens. Of course the code needs tweaking it's not terribly complex, and those files could be pasted into this doc.

As-is, the Flutter tab is drastically more complex than the other languages, and it shouldn't be too hard to change the code and say "follow xyz steps of the flutter tutorial" instead of follow the whole thing.

If this does stay as-is, are there really no changes that need to be made to the robot screen code to enable this navigation? I don't remember off hand, just double checking that that's been tested.

Reply:
You do need to edit the robot screen code, see the second part of this code. 

Re. not going through whole flutter app, I can try but it will take me a while. I'm not as familiar with flutter. I had already built the flutter app for the base move in square tutorial so I used that but tweaked for just the motor code/screens. I think it would be a lot of work to instruct them to build a flutter app entirely in this space but if you think that's the go ahead I can give it a shot. It might help me if you were to share one of those test apps?

Reply:
Ooh yes I see I was looking at the full code section and not realizing, sorry. Okay cool.

So, I don't have a test app ready to share, because I edited mine adding lots of pages for lots of resources when I was testing the methods for the flutter SDK code example snippets. So it's not clean, and might be out of date. I was just figuring things out like you would be; hadn't used Flutter before other than editing/putting together the flutter tutorial based on Clint's doc. But, if you think this is too much work for now would you mind making a ticket to revisit this for this and the square one? I think it's a good idea not just for simplicity of these quickstarts but also to show a different construction of an app. TY!

Reply:
```suggestion
To get started programming your rover with Flutter, follow the instructions to [Build a Flutter App that Integrates with Viam](/tutorials/control/flutter-app/).
Then return to this page to add motor control to your app.
```
Or something to make it clear that they should do that and then continue here.

Reply:
I'm not sure I fully understand - so does this work without it being a rover, i.e. without having a base?

Reply:
it should, the flutter app tutorial doesn't add any rover specific code, just a home screen and robot screen. This was in error, thank you for catching (my eyes confused rover->robot)

Reply:
Sorry, the rover -> machine is not the important part of my comment. The important part is that I think we should say "Then return to this page to add motor control to your app." And yeah, you can 1000% have a motor without it being a rover. This isn't the drive in square quickstart, it's just a motor.